<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>320</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="buttonAddSlot">
      <property name="text">
       <string>Add Slot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonRemoveSlot">
      <property name="text">
       <string>Remove Slot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonOk">
      <property name="text">
       <string>Speichern</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>244</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="teacherNameLabel">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="teacherNameLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableAvailability">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <column>
       <property name="text">
        <string>Day</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Start</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>End</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
